server/inventory_item/updater.py

```diff
@@ -0,0 +1,66 @@
+from apscheduler.schedulers.background import BackgroundScheduler
+from apscheduler.jobstores.mongodb import MongoDBJobStore
+from apscheduler.events import EVENT_JOB_EXECUTED, EVENT_JOB_ERROR
+import pymongo as pym
+from django_server.load_csv_to_db import main
+
+
+# Making connection to mongoclient
+client = pym.MongoClient("mongodb://localhost:27017/")
+
+job_stores = {
+    'default': MongoDBJobStore(database="alta_development",
+                               collection="jobs", client=client)
+}
+
+# This will get you a BackgroundScheduler with a MongoDBJobStore
+# named “default” to replace the default one  and a ThreadPoolExecutor
+# named “default” with a default maximum thread count of 10.
+scheduler = BackgroundScheduler(jobstores=job_stores)
```

Comment:
Yeah, I don't really know how to do that, I suggest to get on a call to explain more your approach. Also I did it this way because i need to access the scheduler globally everywhere